frontend/src/metabase/visualizations/components/TableInteractive/TableInteractive.jsx

```diff
@@ -799,6 +825,8 @@ class TableInteractive extends Component {
         }}
       >
         <HeaderCell
+          c={cellColor}
+          bg="white"
```

Comment:
@WiNloSt I ran into a couple of subtle CSS issues while adding support for colored headers (mainly hover states), so I decided to only support white background for now. I would prefer to support this in later PR because there was quite a few edge cases.